src/Qowaiv/OpenApi/OpenApiDataType.cs

```diff
@@ -53,6 +57,9 @@ public sealed record OpenApiDataType
     public bool Nullable { get; }
 
     /// <summary>Gets the Pattern of the OpenAPI Data Type.</summary>
+#if NET7_0_OR_GREATER
```

Comment:
Inconsistencies between 
#if net70 [attr] prop #else prop #endif
and
#if net70 [attr] #endif prop

I prefer the latter, maybe even on a single line.

Reply:
Another consideration could be to define a "dummy" StringSyntaxAttribute for dotnet 6 and lower. But for these 4 occurrences I wouldn't bother.

Reply:
@Sjaaky I considered the second option. But is both an attribute and a helper static class.

About `#if .+` The compiler does not support any code after the `#` so your suggestions would not compile.